contribs/gnoserve/examples/gno.land/r/gnoserve/index.gno

```diff
@@ -0,0 +1,31 @@
+package gnoserve
+
+var newSource = `<gno-mark>
+
+{
+  "gnoMark": "petrinet",
+  "modelType": "PetriNet",
+  "version": "v1",
+  "tokens": ["black"],
+  "places": {
+    "place0": { "offset": 0, "initial": [1], "capacity": [3], "x": 130, "y": 207 }
+  },
+  "transitions": {
+    "txn0": { "x": 46, "y": 116 },
+    "txn1": { "x": 227, "y": 112 },
+    "txn2": { "x": 43, "y": 307 },
+    "txn3": { "x": 235, "y": 306 }
+  },
+  "arcs": [
+    { "source": "txn0", "target": "place0", "weight": [1] },
+    { "source": "place0", "target": "txn1", "weight": [3] },
+    { "source": "txn2", "target": "place0", "weight": [3], "inhibit": true },
+    { "source": "place0", "target": "txn3", "weight": [1], "inhibit": true }
+  ]
+}
```

Comment:
This approach has been refined a bit in https://github.com/allinbits/gnoserve

Currently it is only required that content be valid Json and that the key "gnoMark" is set.
the GnoMark Markdown Extension uses this value to output html fragments from templates that can be on-chain or JS injected mechanisms.